src/components/slide-toggle/slide-toggle.html

```diff
@@ -0,0 +1,22 @@
+<label class="md-slide-toggle-label">
+  <div class="md-slide-toggle-container">
+    <div class="md-slide-toggle-bar"></div>
+    <div class="md-slide-toggle-thumb-container">
+      <div class="md-slide-toggle-thumb"></div>
+    </div>
+
+    <input #input class="md-slide-toggle-checkbox" type="checkbox"
+           [id]="getInputId()"
+           [tabIndex]="tabIndex"
+           [checked]="checked"
+           [disabled]="disabled"
+           [attr.name]="name"

```

Comment:
This is a good point. I also had that before. But this will cause issues with a falsy value.

For example, if the user, uses `[name]="myName", and the `myName` property is set to `null` will result in
```html
<input name="null">
```